libs/wallet-provider/README.md

```diff
@@ -0,0 +1,32 @@
+# CoW Swap Wallet provider
+
+The library contains only two hooks:
+ - `useWalletChainId()` returns actual wallet chainId. In 99% cases we use chainId from `useWalletInfo()` which returns always valid and supported chainId, but in some cases we need a real chainId from the wallet.
+ - `useWalletProvider()` returns ethers `Web3Provider`.
+
+Those hooks are actually wrappers around `useWeb3React()` from `@web3-react/core`.  
+Once we decide to use other library, we can easily replace it.  
+
+## Quick start
+
+```
+npm install @cowprotocol/wallet-provider --save
+```
+
+```
+yarn add @cowprotocol/wallet-provider
+```
```

Comment:
Do we need this exposed as an npm lib?

Unless we do, we might as well not have these instructions.

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/4420/commits/388ea3a470e812cdfca86a6333dd127a8f861372